### PR TITLE
Add support for Arc browser (macOS only)

### PR DESCRIPTION
**Pull request by EmrysMyrddin**
This PR aims to add make rules for the Arc Browser.

The rules are macOS only since the browser is only available there for now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -208,6 +208,34 @@ hosts-chrome-user:
 	    ;; \
 	esac
 
+.PHONY: hosts-arc
+hosts-arc-user:
```
Comment (maximbaz) [inline Makefile:212]:
```suggestion
hosts-arc:
```

File: Makefile [hunk 2]
```diff
@@ -541,6 +569,34 @@ policies-chromium-user:
 	    ;; \
 	esac
 
+.PHONY: policies-arc
+policies-arc-user:
```
Comment (maximbaz) [inline Makefile:573]:
```suggestion
policies-arc:
```

**Review by maximbaz — COMMENTED** [review]
Amazing, thanks! Just spotted a couple typos, otherwise I trust you have tested it works, so will merge right away :+1:

**Comment by EmrysMyrddin** [comment]
Ah good catch sorry ^^' I've not tested the system wide rules since I don't have my browser installed system wide. But since it works for local installs there is no reason that it doesn't works.

**Comment by EmrysMyrddin** [comment]
In fact, I didn't checked before, but it seems that the extension is not actually working, but it's not related to the native messaging. It's the extension failling to find forms to fill in the page... not sure why yet.

I supposed this can still be merged, the fix will probably be done in the main browserpass repo.

**Comment by EmrysMyrddin** [comment]
Thank you for your work by the way, this is really an amazing extension with a really great UX !

**Comment by maximbaz** [comment]
Fair enough, thanks! 

With regards to the form filling, it is really dependent on specific websites and how "machine-friendly" their login forms are to detect, unless you reproduce the issue on all websites, there's a pinned issue in browserpass-extension to collect all "broken" websites.

**Pull request merged**